src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaItemLabel.tsx

```diff
@@ -49,10 +52,20 @@ export const SchemaItemLabel = (props: SchemaItemLabelProps) => {
     setContextAnchor(e.currentTarget);
     e.stopPropagation();
   };
-  const handleAddPropertyClick = (e: React.SyntheticEvent, type: PropertyType) => {
+  const handleAddNode = (e: React.SyntheticEvent, type: PropertyType) => {
```

Comment:
why have this in between the actual function calls? Seems like you are checking the function twice.